canvas_modules/common-canvas/src/common-properties/controls/abstract-table.jsx

```diff
@@ -455,7 +455,7 @@ export default class AbstractTable extends React.Component {
 					data={rows}
 					rows={0}
 					scrollKey={this.selectSummaryPropertyName}
-					control={this.props.control}
+					tableLabel={this.props.control.label ? this.props.control.label.text : ""}
```

Comment:
Will there always be `text` under `label`? or do we need to also check for `this.props.control.label.text`?